src/containers/BooksList.js

```diff
@@ -0,0 +1,20 @@
+import Book from '../components/Book';
+
+const BooksList = () => (
+    <div>
```

Comment:
**[OPTIONAL]**
The use of fragment in this instance instead of a `div` is more efficient. There is no styling dependent on the `div` hence we can use a fragment so that no extra no would be created. We can change `<div></div>` to `<></>`